bundles/org.openhab.binding.deconz/README.md

```diff
@@ -218,6 +221,7 @@ Bridge deconz:deconz:homeserver [ host="192.168.0.10", apikey="ABCDEFGHIJ" ] {
     presencesensor      livingroom-presence     "Livingroom Presence"       [ id="1" ]
     temperaturesensor   livingroom-temperature  "Livingroom Temperature"    [ id="2" ]
     humiditysensor      livingroom-humidity     "Livingroom Humidity"       [ id="3" ]
+    airqualitysensor    livingroom-voc          "Livingroom Voc"            [ id="4" ]
     pressuresensor      livingroom-pressure     "Livingroom Pressure"       [ id="4" ]
```

Comment:
Do you need to renumber id's as there are now two with `id="4"`?

Reply:
I have assigned a new unique id for the example.